adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Microsoft.IdentityModel.Clients.ActiveDirectory.csproj

```diff
@@ -9,7 +9,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <TargetFrameworks>net45;netstandard1.1;netstandard1.3;MonoAndroid8.1;Xamarin.iOS10</TargetFrameworks>        
+    <TargetFrameworks>net45;netstandard1.1;netstandard1.3;win81;uap10.0;MonoAndroid8.1;Xamarin.iOS10</TargetFrameworks>        
```

Comment:
As a suggestion / future enhancement - there seem to be quite a few properties and item groups that are shared between all the projects - these can be consolidated in a common.targets file, making the individual csproj easier to read.

Reply:
that's agreat update for a future PR. I already re-grouped in a few previous PR's. Feel free to make additional changes, you can add those to this PR/branch if you like.